### ISSUE TITLE
syntax error not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2555]
should give error for

```abap
DATA lt_tadir TYPE STANDARD TABLE OF tadir WITH DEFAULT KEY.
SELECT * FROM tadir INTO TABLE lt_tadir ORDER BY obj_name UP TO 100 ROWS.
```

![image](https://user-images.githubusercontent.com/5888506/170815224-4eaf1bea-5b20-4762-a764-e3e31b39e7b2.png)
